src/style.css

```
@@ -0,0 +1,120 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
  margin: 0;
}
```
check how the `*` selector works! [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/Universal_selectors)